notices.json

```diff
@@ -1,5 +1,23 @@
 [
-{
+    {
+        "id": "focalboard_1",
+           "conditions": {
+          "audience": "sysadmin",
+          "clientType": "all",
+          "serverVersion": [">5.35"],
+          "displayDate": ">= 2021-06-16T00:00:00Z"
+        },
+        "localizedMessages": {
+          "en": {
+            "title": "Enable Focalboard",
+            "description": "Focalboard integrates project management tools like kanban boards, cards, and tables with Mattermost to help you easily plan projects, manage tasks, and run effective meetings. Enable Focalboard today following the steps in the link below.",
+            "image": "https://user-images.githubusercontent.com/46905241/121941290-ee355280-cd03-11eb-9b9f-f6f524e4103e.gif",
+            "actionText": "Learn more",
+            "actionParam": "https://www.focalboard.com/download/mattermost"
```

Comment:
I think this would really help lower the barrier to entry here: https://mattermost.atlassian.net/browse/MM-36441

Reply:
This would be nice-to-have, but moot for Focalboard once we are enabled by default.

Reply:
Ah. If that's the case curious why the text is asking admins to follow the steps to enable focalboard? Sorry, I might be missing something here. @chenilim

Reply:
The Focalboard websocket URL needs to be added to the web-proxy (nginx, Apache) config (see [thread](https://community.mattermost.com/core/pl/96d1hz9o7fdrzdfh7ettrjcbzo)). Our mitigation is:
* Show an [error banner](https://github.com/mattermost/focalboard/pull/570) with a link to the docs on websocket errors, and
* (In the next release) Add a [link to docs](https://focalboard-community.octo.mattermost.com/workspace/qgsck6cts3fwpqwyjiupjm5cde/shared?id=47aa9bb4-6967-4a96-83c7-11bd6b20f1eb&v=ca1a5441-4c1c-4d0e-856e-88cc744576c8&r=762db9c6-305c-4d13-8ce6-9f11e5232c24&c=a67e26c8-4e40-4c86-8a5a-c336f36bcf99) in the plugin config screen.

Reply:
Got it, thanks for the context